### ISSUE TITLE
[Question/Enhancement] Static image duration

### ISSUE THREAD

**Issue by MickaelHrndz** [alexmercerind/media_kit#277]
Hello,

I have to show a playlist of videos and pictures mixed together. I'm glad the package support showing png/jpg files, but right now the images show about a second and disappear.
Is it possible to set the duration ? And if not, would you be willing to integrate that option ? Or should I use some other package to convert the image to a GIF/video ?

**Comment by alexmercerind** [comment]
**TL;DR**

Please do not load images with package:media_kit.

<hr>

Hi!

Even though currently images load just fine (thanks to available image decoders/demuxers in the FFmpeg), it is going to be removed in the next version. Internally, I'm focusing on reducing the bundle size (especially on mobile platforms):

At the moment (referring to v1.0.0), package:media_kit (video variant) weighs around 10 MB for Android ARM64 (includes very wide codec/format support). I was able to bring it under 5 MB:

![image](https://github.com/alexmercerind/media_kit/assets/28951144/bcc9dac5-77b7-4bb5-a211-0f238433edc6)

These changes will be shipped in next update. The size is further decreased for audio variant.

Flutter itself has good support for rendering/decoding most image formats; it shouldn't take much to write own `Widget` to display both images & videos. 

<hr>

Still... you have the choice to manually replace the dynamic libraries etc. provided by `media_kit_libs_*` yourself. You can choose to use a more capable version of libmpv/FFmpeg which has better codec/format support (& thus, a larger size).

For now, project's focus is common applications that rely on commonly used video/audio formats. In future, we may provide `media_kit_libs_video_full_*` & `media_kit_libs_audio_full_*` etc. At the moment, it is too much to take care of & short-term goals are prioritized. Further improving the performance, stability & features.

Thanks!

**Comment by alexmercerind** [comment]
The bundle size is a big concern for mobile applications. I consider anything below 30 MB good; still, we're making a further decrease in the size of the package. I hope this answers your question.

**Comment by alexmercerind** [comment]
In case you manually use a version of libmpv/FFmpeg with image support, you can use [`image-display-duration`](https://mpv.io/manual/stable/#options-image-display-duration) property to increase/decrease the display duration. We offer an API to set backend specific attributes:

```dart
final player = Player();
if (player.platform is libmpvPlayer) {
  await (player.platform as dynamic).setProperty('image-display-duration', '69');
}
```

This is not part of the "stable API".

**Comment by MickaelHrndz** [comment]
Thanks for the detailed answers ! 
I can understand that you prioritize reducing size and optimizing. The `_full` variants could be great for use cases where you want to support the most codecs/formats you can, VLC-style.
Anyways, I think my specific case calls for converting those images to videos.
